acceptance/testdata/launcher/container/layers/config/metadata.toml

```diff
@@ -0,0 +1,13 @@
+[[buildpacks]]
+  id = "some/buildpack"
+  version = "0.0.1"
+
+[[processes]]
+  type = "web"
+  command = "echo \"Executing web process-type\""
+  direct = false
```

Comment:
It doesn't seem that we are testing any `direct = true` cases, is that true? Would that be something worth adding?

Reply:
We are executing `direct = true` process in some of the tests where we override the entire command, but yes I'll change one of these buildpack-provided processes.